src/EditorFeatures/Core/Classification/Syntactic/SyntacticClassificationTaggerProvider.TagComputer.cs

```diff
@@ -358,17 +358,17 @@ async ValueTask<SnapshotSpan> ComputeChangedSpanAsync()
                     return _lastProcessedData;
             }
 
-            public IEnumerable<ITagSpan<IClassificationTag>> GetTags(NormalizedSnapshotSpanCollection spans)
+            public SegmentedList<ITagSpan<IClassificationTag>> GetTags(NormalizedSnapshotSpanCollection spans)
```

Comment:
made things more strongly typed.  Allows for the consumption side (still within roslyn), to do things like non-allocate enumerate through things.